src/styles/block/main.scss

```diff
@@ -0,0 +1,138 @@
+:root {
+  --main-columns-number: 1;
+}
+
+.main {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  display: grid;
+  grid-template-columns: repeat(var(--main-columns-number), 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  padding: 50px 40px;
+  gap: 48px 46px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  :root {
+    --main-columns-number: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  :root {
+    --main-columns-number: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  :root {
+    --main-columns-number: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  padding-inline: 16px;
+
+  transition: transform $transition-time;
+
+  &__image {
+    width: 160px;
+    padding-top: 32px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+  &__text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__button {
+    width: 160px;
+    height: 40px;
+    margin-top: 16px;
+    color: #fff;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: none;
+    cursor: pointer;
+  }
+
+  &__button:hover {
+    color: #00acdc;
+    background-color: #fff;
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
+}
+
+.rating {
+  &__block {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    gap: 4px;
+
+    &__text {
+      font-size: 10px;
+      font-weight: 400;
+      color: #060b35;
+      line-height: 14px;
+    }
+  }
+
+  &__stars {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-size: contain;
```

Comment:
The `background-size: contain;` property here ensures that the star image is scaled to fit within its container without changing its aspect ratio. However, if you want the star to cover the entire area of the container, consider using `background-size: cover;`. Make sure the chosen property aligns with the design requirements.